lib/SILGen/SILGen.cpp

```diff
@@ -1373,6 +1373,9 @@ SILFunction *SILGenModule::emitClosure(AbstractClosureExpr *ce) {
   if (!f->isExternalDeclaration())
     return f;
 
+  // Emit property wrapper argument generators.
+  emitArgumentGenerators(ce, ce->getParameters());
```

Comment:
Heh, building SwiftPM with this change crashes here. I assumed that closure parameters never have default arguments, but that's not true. Here's a case where the type-checked AST has an autoclosure with a default argument:

```swift
enum E {
  struct Result {
    var name: String
    var platforms: [Int] = []
  }

  static func test() {
    var result = Self.Result(name: "") // the call to Self.Result.init gets wrapped in an autoclosure, and immediately called
  }
}
```

SILGen crashes when attempting to emit the default argument generator here because it tries to get an enclosing declaration for the parameter, which doesn't exist here.